src/EOS.hpp

```diff
@@ -25,8 +26,8 @@
 
 namespace quokka
 {
-static constexpr double boltzmann_constant_cgs = 1.380658e-16; // cgs
-static constexpr double hydrogen_mass_cgs = 1.6726231e-24;     // cgs
+static constexpr double boltzmann_constant_cgs = C::k_B;     // cgs
+static constexpr double hydrogen_mass_cgs = C::m_p + C::m_e; // cgs
```

Comment:
`hydrogen_mass_cgs` should not be defined. We don't need it for the EOS.

Reply:
So, wherever we are using the hydrogen mass, we replace it by the atomic mass? @BenWibking

Reply:
Yes, I think so. There might be one or two exceptions, but probably easier to just change it everywhere and see if any of the tests fail.

Reply:
Actually yes, I think it should be changed everywhere. I thought the opacity was defined by H atom somewhere, but I couldn't find it.